### ISSUE TITLE
'>=" and "<=" are parsed/formatted incorrectly

### ISSUE THREAD

**Issue by andr2k** [anboralabs/intellij-firebase-highlighter#116]
**Describe the bug**
'>=" and "<=" are parsed/formatted incorrectly

**To Reproduce**
```
            allow create, update: if (request.auth != null &&
            request.auth.uid == userId &&
            request.resource.data.keys().hasOnly(['firstName', 'middleName', 'lastName', 'birthDate', 'phone']) &&
            request.resource.data.firstName is string &&
            request.resource.data.firstName.size() >= 1 &&
            request.resource.data.middleName is string &&
            request.resource.data.lastName is string &&
            request.resource.data.lastName.size() >= 1);
```

**Result**

<img width="1013" alt="image" src="https://github.com/anboralabs/intellij-firebase-highlighter/assets/922393/3f54870a-cdbe-4839-884f-35bee2644aec">

**Expected**

<img width="991" alt="image" src="https://github.com/anboralabs/intellij-firebase-highlighter/assets/922393/8dfce734-d701-4578-8dbd-f488546992dd">

**Comment by dalgarins** [comment]
Issue cloned to https://github.com/anboralabs/intellij-firebase/issues/63

**Comment by dalgarins** [comment]
@andr2k the issue is fixed and it will be available on version:  2023.4.8
<img width="841" alt="Screenshot 2023-07-07 at 4 48 22 PM" src="https://github.com/anboralabs/intellij-firebase-highlighter/assets/8527174/4e760cba-b63b-4742-a771-3c982afbd04c">
